aspnet/mvc/views/forms/sample/final/ViewModels/AddressViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace FormsTagHelper.ViewModels
+{
+    public class AddressViewModel
+    {
+        public string AddressLine1 { get; set; }
+    }
+
+    public class RegisterAddressViewModel
```

Comment:
@dougbu  I thought it made it easier to follow the document

![image](https://cloud.githubusercontent.com/assets/3605364/14445246/62f40502-0000-11e6-817b-b324ca49d5cb.png)

I've spit it out - good programming practice trumps imaginary ease of reading code associated with document.